transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java

```diff
@@ -497,10 +501,16 @@ private String generateXmlModel(XmlModelGenerator generator,
             Iterator<JDefinedClass> classIter = modelPkg.classes();
             if (classIter.hasNext()) {
                 JDefinedClass modelClass = classIter.next();
+                IType foundType = project.findType(modelClass.fullName(), monitor);
                 IPackageFragment newPkg = project.findType(modelClass.fullName(), monitor).getPackageFragment();
                 String newPkgName = newPkg.getElementName() + '_' + time;
-                newPkg.rename(newPkgName, true, monitor);
+                
+                // handle refactoring via the refactor framework so we get package renaming
```

Comment:
I think that this comment should be part of the commit message, not the code.
When reading code we see that it is using the Refactoring Suppport.